src/org/exist/storage/DBBroker.java

```diff
@@ -116,6 +116,8 @@
 
     protected IndexController indexController;
 
+    long config_timestamp = 0;
```

Comment:
config_timestamp is never updated?

Reply:
yeah, that is typo that pop up on  refactoring. Fixed.